unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneBoundariesController/SceneBoundariesChecker.cs

```diff
@@ -53,7 +108,7 @@ void EvaluateMeshBounds(DecentralandEntity entity)
             Bounds meshBounds = Utils.BuildMergedBounds(entity.meshesInfo.renderers);
```

Comment:
Optimization idea here: We could store the meshBounds here and if only position has changed just add the difference, only recalculating it if rotation/scaling changes.